js/ui/overlay/ui.overlay.js

```diff
@@ -100,6 +100,10 @@ var getElement = function(value) {
     return value && $(value.target || value);
 };
 
+function isPercent(size) {
+    return !!size && (size.toString().slice(-1) === '%');
```

Comment:
Try to use "../core/utils/size"

Reply:
There is no enough functionality, so I supplemented it